ttml/profile/dapt/Overview.html

```diff
@@ -11,7 +11,6 @@ <h1></h1>
 
 <pre>
 http://www.w3.org/ns/ttml/profile/dapt#metadata
-http://www.w3.org/ns/ttml/profile/dapt/extension/
```

Comment:
Presumably we need a file at this path though?

Reply:
there is
https://github.com/himorin/w3c-ns/blob/ttwg-dapt/ttml/profile/dapt/extension/Overview.html

Reply:
Thanks, I hadn't spotted that.